ckan/controllers/group.py

```diff
@@ -227,15 +240,15 @@ def _read(self, id, limit):
         def search_url(params):
             if group_type == 'organization':
```

Comment:
@davidread shouldn't this be checking is_organization instead? I ask because I see you fixed some code near this